lm_eval/filters/extraction.py

```diff
@@ -62,11 +62,8 @@ def apply(self, resps, docs):
         def filter_set(inst):
             filtered_resp = []
             for resp in inst:
-                if resp.startswith(" "):
-                    resp = resp[1:]
-
+                resp = resp.lstrip()
```

Comment:
there can be other whitespaces and there can be multiple ones so better to use `lstrip()`